.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "loopback",
+  "rules": {
+    "max-len": ["error", 100, 4, {
+      "ignoreComments": true,
+      "ignoreUrls": true,
+      "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
+    }]
+  }
+}
```

Comment:
On the second thought, I think JSON is better here. Compare:

```json
{
  "extends": "loopback",
  "rules": {
    "max-len": ["error", 100, 4, {
      "ignoreComments": true,
      "ignoreUrls": true,
      "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
    }]
  }
}
```

with YAML

```yaml
---
  extends: "loopback"
  rules:·
    max-len:·
      - "error"
      - 100
      - 4
      -
        ignoreComments: true
        ignoreUrls: true
        ignorePattern: "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
```

Thoughts?